ceagle/api/v1/status.py

```diff
@@ -29,9 +32,17 @@ def get_status(period):
 
 
 @bp_status.route("/health/<period>")
-@fake_status.get_status_health
 def get_status_health(period):
-    return flask.jsonify("fixme!")
+    health_client = client.get_client("health")
+    if not health_client:
+        return flask.make_response(
+            json.dumps({"error": "No health endpoint configured"}),
+            500,
+        )
+
+    api_endpoint = "/api/v1/health"
```

Comment:
for example health api has '/api/v1/regions' endpoint — if you put "http://url:port/api/v1/health" in the health endpoint — there is no way to get regions api endpoint from health.